crates/elements/src/definitions.rs

```diff
@@ -479,6 +479,8 @@ builder_constructors! {
     /// }
     /// ```
     svg {
+        #[doc = include_str!("_docs/attributes/color.md")]
```

Comment:
You will need to create the doc file as well
```suggestion
        #[doc = include_str!("_docs/attributes/fill.md")]
```